### DISCUSSION TITLE
Hi, a few questions, with your permission?

### DISCUSSION THREAD

**Discussion by Soldream0** [Arisego/UnrealLive2D#19] [Q&A] [answered]
I'm new to UE and I'm about to get into game development using Blueprints. I'm serious about it. 

1. I have seen several Reports of problems with plugin compilation. What do you think these errors are related to?
2. I would like to use live2d in UE. What version of UE do you recommend for the most stability? And maybe there are some things to avoid with this plugin?
3. how stable is your plugin and is it possible to make a full project with it? I mean, is it mostly abandoned or is it mostly finished and ready?
4. If this question is appropriate based on past answers and questions. Perhaps you could recommend an alternative to live2d for UE?

**Comment by Arisego** [comment, marked as answer]
Hello, this plugins is not production ready, it's just a starting point for integrating live2d into unreal. So it requires a bit of coding experiment to make it work. And it was not supposed to be used in a blueprint project, cause you may need some functions that is still in Live2D SDK. 

As I haven't been actively working with Unreal Engine lately, I cannot guarantee the stability of specific versions. Nevertheless, released versions generally exhibit stability with only minor bugs.

If you're seeking alternatives to Live2D, you may want to take a look at Spine, which offers official Unreal Engine integration and a lot of 2D game used it too.

**Reply by Soldream0** [reply]
Thanks, I'll check out Spine.
I guess as a newbie I shouldn't start trying to use your plugin? By the way, besides Spine, are there no other similar plugins? I hope I'm not asking too many questions.

**Reply by Arisego** [reply]
Yeah, I don't recommend you use this plugin if you are new to unreal engine c++, you may fall into some bad coding problems that consumes a lot of time. 

For 2D animation solution, I don't have too much knowledge about it. I know Spine2D just because one of my favorite game used it. So I am not able to provide any good advises on this topic.
